test/src/e2e_vm_tests/test_programs/should_pass/language/configurable_consts/src/main.sw

```diff
@@ -41,6 +41,12 @@ configurable {
     ARRAY_U64: [u64; 3] = [9, 8, 7],
     TUPLE_BOOL_U64: (bool, u64) = (true, 11),
     STR_4: str[4] = __to_str_array("abcd"),
+    VEC_U8: Vec<u8> = Vec::new(),
+    STRING_SLICE: str = "Hello, Sway",
+
+    // The test runner does not have a way to calculate the dynamic part of the data section
```

Comment:
Do we want to add that to the test runner as a separate issue?

Reply:
I think it makes sense. I aggregated other suggestions at https://github.com/FuelLabs/sway/issues/6866.